Nav.php

```diff
@@ -202,6 +202,32 @@ public function renderItem($item)
     }
 
     /**
+     * Check if parent item is visible
+     * @param array $parentItem
+     * @return boolean
```

Comment:
Maybe `2.0.6`? This extension has no version `2.0.6.` released yet and has own version history as I see.